dpcpp/CMakeLists.txt

```diff
@@ -75,7 +78,7 @@ else ()
     target_link_options(ginkgo_dpcpp PUBLIC -fsycl-device-code-split=per_kernel)
 endif()
 target_link_libraries(ginkgo_dpcpp PUBLIC ginkgo_device)
-target_link_libraries(ginkgo_dpcpp PRIVATE MKL::MKL_DPCPP)
+target_link_libraries(ginkgo_dpcpp PRIVATE MKL::MKL_DPCPP oneDPL)
```

Comment:
Are we using this already? Or is this preparation for IDR? Maybe leave it for a later PR then?

Reply:
good catch. Yes, it is the IDR related thing. I will delete them